### PR TITLE
Integrated Mayhem

### PR DESCRIPTION
**Pull request by bmwtsn098**

### CODE REVIEW

File: Mayhemfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Namespaced project name that the target belongs to
```
Comment (xansec) [inline Mayhemfile:1]: Please omit image and duration, and use a generic project name.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by bmwtsn098** [comment]
I've made the requested changes. Let me know if everything looks good. Thanks

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**